config/routes.rb

```diff
@@ -9,6 +9,10 @@
   resources :orders, only: [:index, :create, :show]
   get "/dashboard", to: "users#show", as: "dashboard"
 
+  namespace :admin do
+    resources :dashboard, only: [:index], as: "user"
```

Comment:
This was something I changed while tests were breaking - never thought to change it back. After fiddling with it, it looks like you don't need that bit. Changed it and changed the path helper in the one test where it's called.